ansible/roles/test/files/ptftests/inner_hash_test.py

```diff
@@ -479,17 +493,15 @@ def check_balancing(self, dest_port_list, port_hit_cnt):
             total_entry_hit_cnt = 0
             for member in ecmp_entry:
                 total_entry_hit_cnt += port_hit_cnt.get(member, 0)
-            (p, r) = self.check_within_expected_range(total_entry_hit_cnt, float(total_hit_cnt)/len(dest_port_list))
+
+            total_expected = float(total_hit_cnt) / len(dest_port_list)
+            if self.symmetric_hashing and hash_key != 'ip-proto':
+                total_expected = total_expected * 2
+
+            (p, r) = self.check_within_expected_range(total_entry_hit_cnt, total_expected)
             logging.info("%-10s \t %-10s \t %10d \t %10d \t %10s"
-                        % ("ECMP", str(ecmp_entry), total_hit_cnt//len(dest_port_list), total_entry_hit_cnt, str(round(p, 4)*100) + '%'))
+                        % ("ECMP", str(ecmp_entry), (total_hit_cnt//len(dest_port_list)*len(ecmp_entry)), total_entry_hit_cnt, str(round(p, 4)*100) + '%'))
             result &= r
-            if len(ecmp_entry) == 1 or total_entry_hit_cnt == 0:
-                continue
-            for member in ecmp_entry:
-                (p, r) = self.check_within_expected_range(port_hit_cnt.get(member, 0), float(total_entry_hit_cnt)/len(ecmp_entry))
-                logging.info("%-10s \t %-10s \t %10d \t %10d \t %10s"
-                            % ("LAG", str(member), total_entry_hit_cnt//len(ecmp_entry), port_hit_cnt.get(member, 0), str(round(p, 4)*100) + '%'))
-                result &= r
```

Comment:
@anish-n this section has been removed, since the validation of equivalent distribution within the LAG members won't be working for inner 5-tuple. As previously discussed, we are focusing only on LAG/PORT ECMP. If any concerns - please let us know.